mod/default-config.json

```diff
@@ -1,56 +1,62 @@
-    "Fuel Capacity: % Increase Per Upgrade": "50",
-
-    "Oxygen Capacity: Initial % of Vanilla": "50",
-    "Oxygen Capacity: % Increase Per Upgrade": "50",
-
-    "Boost Duration: Initial % of Vanilla": "50",
-    "Boost Duration: % Increase Per Upgrade": "50",
-
-    "Separator - Quality of Life": {
-        "type": "separator"
-    },
-
-    "Auto Expand Nomai Text": {
-        "type": "toggle",
-        "value": false,
-        "tooltip": "With this on, all Nomai text in the game will start in an expanded state, allowing you to read it without having to wait for it to expand. This setting will only apply at the start of the next loop."
-    },
-
-    "Separator - Crash Testing": {
-      "type": "separator"
-    },
-
-    "[DEBUG] Disable In-Game Console": false,
-    "[DEBUG] Don't Send Locations In-Game": false,
-    "[DEBUG] Don't Receive Items In-Game": false,
-    "[DEBUG] Don't Write To Save File In-Game": false
-  }
+    "settings": {
+        "AP Console: Mute": false,
+        "AP Console: About Me Filter": false,
+
+        "Separator - Quality of Life": {
+            "type": "separator"
+        },
+
+        "Auto Expand Nomai Text": {
+            "type": "toggle",
+            "value": false,
+            "tooltip": "With this on, all Nomai text in the game will start in an expanded state, allowing you to read it without having to wait for it to expand. This setting will only apply at the start of the next loop."
+        },
+
+        "Instant Translator": {
+            "type": "toggle",
+            "value": false,
+            "tooltip": "With this on, your translator will translate Nomai Text instantly."
+        },
+
+        "Separator - Death Link": {
+            "type": "separator"
+        },
+
+        "Death Link Override": {
+            "type": "selector",
+            "value": "No Override",
+            "options": [
+                "No Override",
+                "Off",
+                "Default",
+                "All Deaths"
+            ],
+            "tooltip": "'No Override' means use the death_link option from the .yaml used to generate the multiworld. Any other value will override and ignore the .yaml option."
+        },
+
+        "Separator - Suit Upgrades": {
+            "type": "separator"
+        },
+
+        "Fuel Capacity: Initial % of Vanilla": "50",
+        "Fuel Capacity: % Increase Per Upgrade": "50",
+
+        "Oxygen Capacity: Initial % of Vanilla": "50",
+        "Oxygen Capacity: % Increase Per Upgrade": "50",
+
+        "Boost Duration: Initial % of Vanilla": "50",
+        "Boost Duration: % Increase Per Upgrade": "50",
+
+        "Separator - Crash Testing": {
+            "type": "separator"
+        },
+
+        "[DEBUG] Disable In-Game Console": false,
+        "[DEBUG] Don't Send Locations In-Game": false,
+        "[DEBUG] Don't Receive Items In-Game": false,
+        "[DEBUG] Don't Write To Save File In-Game": false
+    }
 }
```

Comment:
was the indentation change intentional? not that I have a strong opinion here, but the inconsistency makes it look like an overzealous autoformatter

Reply:
Was likely just VS doing it, I didn’t manually change any formatting

Reply:
Alright it was tedious but I reverted the indentation